sdk/core/core-auth/README.md

```diff
@@ -26,8 +26,8 @@ The `AzureSASCredential` is a static signature-based credential that supports up
 
 ### AzureKeyCredential
 
-```js
-const { AzureKeyCredential } = require("@azure/core-auth");
+```ts snippet:azure_key_credential
+import { AzureKeyCredential } from "@azure/core-auth";
```

Comment:
I know things may have changed but at one point I think we were still adamant about using CJS in samples until ESM is supported in node without any flags. Have we changed that guidance? (not a leading question just making sure I know the latest!)

Reply:
These are by default ESM packages and not to be used by consumers directly as most are re-exported by their other packages.

Reply:
Sure, but what about leaf packages (when we get to them)?